content/concept-amendments.md

```diff
@@ -86,13 +86,122 @@ When an amendment gets enabled for the network after the voting process, servers
 * Do not participate in the consensus process
 * Do not vote on future amendments
 
-Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
+Becoming amendment blocked is a security feature to protect backend applications. Rather than guessing and maybe misinterpreting a ledger after new rules have been applied, `rippled` reports that it does not know the state of the ledger because it does not know how the amendment works.
 
 The amendments that a `rippled` server is configured to vote for or against have no impact on whether the server becomes amendment blocked. A `rippled` server always follows the set of amendments enabled by the rest of the network, to the extent possible. A server only becomes amendment blocked if the enabled amendment is not included in the amendment definitions compiled into the server's source code -- in other words, if the amendment is newer than the server.
 
 If your server is amendment blocked, you must [upgrade to a new version](tutorial-rippled-setup.html#updating-rippled) to sync with the network.
 
 
+#### How to Tell If Your `rippled` Server is Amendment Blocked
+
+***TODO: Question: Okay location for this doc? It is building upon a concept, but is tutorial-ish.***
```

Comment:
It's probably fine to put it here for now.

Long-term, I think this'll fit into section VI.3 of the rippled doc plan ("The rippled Server - Advanced Configuration")

Nitpick: capitalize "is" or don't capitalize "if". I think the former is proper style, although I'm no expert.

Reply:
Here's what the _Microsoft Manual of Style says about title capitalization:

>* Capitalize all nouns, verbs (including is and other forms of be), adverbs (including than and when), adjectives (including this and that), and pronouns (including its).
>* Capitalize the first and last words, regardless of their parts of speech (“The Text to Look For”).
>*  Capitalize prepositions that are part of a verb phrase (“Backing Up Your Disk”).
>*  Do not capitalize articles (a, an, the) unless the article is the  first word in the title.
>*  Do not capitalize coordinate conjunctions (and, but, for, nor, or, yet, so) unless the conjunction is the first word in the title.
>*  Do not capitalize prepositions of four or fewer letters unless the preposition is the first or the last word in the title.
>*  Do not capitalize to in an infinitive phrase (“How to Format Your Hard Disk”) unless the phrase is the first word in the title.
>*  Capitalize the second part of a hyphenated compound if it would be capitalized without the hyphen. Always capitalize the second part of a hyphenated compound if it is the last word of a heading or title.

Reply:
I am no expert either, but from what I've read, 'if" is a subordinating conjunction and should be capitalized -- per AP and Chicago, for example. Sound okay? What is the next style guide in line when Microsoft Manual of Style doesn't specify something?

Reply:
The Microsoft one is more specific than most other style guides I saw when it comes to title case. It seems like most style guides—contrary to what I would've expected—suggest you should capitalize "Is" though.

Reply:
IBM Style Guide on subordinating conjunctions: In headline-style capitalization, capitalize the initial letter of the following words ... subordinating conjunctions, such as after, although, because, before, how, if, than, that, though, until, when, where, whether, and while.